webpack/assets/javascripts/react_app/components/common/table/actions/getTableItemsAction.js

```diff
@@ -6,10 +6,11 @@ import createTableActionTypes from '../actionsHelpers/actionTypeCreator';
  * An async Redux action that fetches and stores table data in Redux.
  * @param  {String} controller the controller name
  * @param  {Object} query      the API request query
+ * @param  {String} plugin     the foreman plugin name
  * @return {Function}          Redux Thunk function
  */
-const getTableItemsAction = (controller, query) => dispatch => {
-  const url = new URI(`/api/${controller}`);
+const getTableItemsAction = (controller, query, plugin = null) => dispatch => {
```

Comment:
`controller` is stored as a key in the state, how about transforming it to a key (i.e in HW model table it can be `modelTable`)